reana_server/status.py

```diff
@@ -638,12 +647,22 @@ def get_job_health(self):
 
         running = len(jobs_status_obj.get_k8s_jobs_by_status("Running"))
         pending = len(jobs_status_obj.get_k8s_jobs_by_status("Pending"))
-        total = running + pending
-        percentage = ClusterHealth.get_percentage(pending, total)
+        used = running + pending
+        total_cluster_memory = NodesStatus().get_total_memory()
+        jobs_memory_limit = REANA_KUBERNETES_JOBS_MEMORY_LIMIT_IN_BYTES
+        total = (
+            round(total_cluster_memory / jobs_memory_limit)
+            if total_cluster_memory and jobs_memory_limit
+            else used
+        )
```

Comment:
I think this logic to calculate "job slots" can be moved to `JobsStatus`.

Some discussion is necessary to see how to handle situations when we run jobs that have k8s limit < `REANA_KUBERNETES_JOBS_MEMORY_LIMIT_IN_BYTES`, as in that case we might end up with more `used` than `total`. cc/ @tiborsimko

Reply:
Thanks for a nice suggestion, moved it to the `JobsStatus`.

regarding custom k8s limit set by users I guess there won't be that much of a problem, since if our calculated slot number would be exceeded, it would simply be shown that 0 slots are available as in this example (8 total slots):
![Screenshot from 2021-12-07 15-48-07](https://user-images.githubusercontent.com/4990025/145052160-405f2495-b6c4-4416-957b-f1016863074d.png)

Reply:
We can be a bit approximative here. If our default job slot is 3 Gi, and users ask for 6Gi, this would eat 2 slots. But if they ask for 4 Gi, we cannot use integer numbers anymore. This plot is basically just an approximation of what happens with "default" job slot settings. When users ask for "more" or "less", perhaps the graph is not fully exact, but I guess we can accept some approximations here. 

One problem is how these "10 available" could be interpreted by the users. If we show absolute numbers, i.e. instead of 10 jobs available we show 30 Gi for jobs available, then things might be clearer. In this way, if a workflow only needs 1Gi job instead of the default 3Gi job, each user could calculate how much is "consumable" for their special needs. Ditto when they need 9 Gi for a job. So OT1H showing absolute numbers with units included might be actually preferable. But OTOH this could mean changing graph logic quite some... Perhaps a simple way could be to display "units = 3 Gi" somewhere near legend.

Reply:
Leaving it as it is as it represents an approximation. We can revisit it in the future depending on the real user's experience.